Makefile

```diff
@@ -85,6 +111,121 @@ prebuild-includes:
 $(GH_PAGES)::
 	@echo "doc available at --> $(DOCS_URL)"
 
+
+# virutalenv setup
+# ----------------
+
+quiet_cmd_pyenv  = PYENV     usage: $ $(PY_ENV_ACT)
+      cmd_pyenv  = \
+	if [ -d "./$(PY_ENV)" -a -x "./$(PY_ENV_BIN)/python" ]; then \
+		echo "PYENV     using virtualenv from $(PY_ENV)"; \
+	else \
+		rm -f $(PY_ENV)/requirements.sha256; \
+		$(PYTHON) -m venv $(VTENV_OPTS) $(PY_ENV); \
+	fi
```

Comment:
Here pyenv does one thing: install the virtualenv, nothing more.

It is possible to check if `$(PYTHON)` exists, but anyway the command will fail.